examples/SX127x/SX127x_Receive_FHSS/SX127x_Receive_FHSS.ino

```diff
@@ -0,0 +1,122 @@
+/*
+   RadioLib SX127x Transmit with Frequency Hopping Example
+
+   This example transmits packets using SX1278 LoRa radio module.
+   Each packet contains up to 256 bytes of data, in the form of:
+    - Arduino String
+    - null-terminated char array (C-string)
+    - arbitrary binary data (byte array)
+
+   Other modules from SX127x/RFM9x family can also be used.
+
+   For default module settings, see the wiki page
+   https://github.com/jgromes/RadioLib/wiki/Default-configuration#sx127xrfm9x---lora-modem
+
+   For full API reference, see the GitHub Pages
+   https://jgromes.github.io/RadioLib/
+
+   The SX1276 / 7 / 8 / 9 supports FHSS or Frequency Hopping Spread Spectrum.
+   Once a hopping period is set and a transmission is started the radio
+   will begin triggering interrupts every hop period where the radio frequency
+   is changed to the next channel. This allows a simple mechanism to abide by
+   the FCC 400ms max dwell time rule.
+   https://www.govinfo.gov/content/pkg/CFR-2019-title47-vol1/pdf/CFR-2019-title47-vol1-sec15-247.pdf
+*/
+
+#include <RadioLib.h> //Click here to get the library: http://librarymanager/All#RadioLib
```

Comment:
Probably not necessary to add the link.

Reply:
It seems silly at first but we've found users cut and paste examples together and often don't document which library they used. The link provides a cheap and easy way for users to get the correct lib quickly.